app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="eight wide column">
+      <h2 class="ui header">
+        {{t 'Discount Codes'}}
+      </h2>
+    </div>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      <button class="ui teal button">
+        <i class="plus icon"></i>
+        {{t 'Create a discount'}}
+      </button>
+    </div>
+  </div>
+  <div class="row">
+    <div class="ui hidden divider"></div>
+    <div class="ui {{if device.isMobile 'vertical'}} basic buttons">
+      <div class="ui button active">
+        {{t 'All'}}
+      </div>
+      <div class="ui button">
+        {{t 'Active'}}
+      </div>
+      <div class="ui button">
+        {{t 'Inactive'}}
+      </div>
+    </div>
+    <table class="ui very basic table">
```

Comment:
@anu-007 check this

**Before**
![screenshot from 2017-06-09 19-25-43](https://user-images.githubusercontent.com/12807846/26978590-ae61b628-4d49-11e7-91bb-3aa7686afdbc.png)

**After**

![screenshot from 2017-06-09 19-26-12](https://user-images.githubusercontent.com/12807846/26978598-b72e03d8-4d49-11e7-89f7-ab1fd62c2456.png)